sass/give.scss

```diff
@@ -1,3 +1,5 @@
+@import 'colors';
+@import 'typography';
```

Comment:
At some point should give.scss share more of style.scss do you think? Not asking a change in this PR. Just trying to understand direction.

Reply:
Let's aim to do that as part of the v2 embed work: https://github.com/findthemasks/findthemasks/issues/649